### PR TITLE
ADDED OPENBSD 7.2

### PR DESCRIPTION
**Pull request by jimh-gh**
OpenBSD 7.2 build successfully.

### CODE REVIEW

**Comment by jimh-gh** [comment]
this built openbsd 7.2 successfully.

**Comment by jimh-gh** [comment]
OpenBSD 7.3 now builds successfully. 
Had to change the cd/iso controller for virtualbox from ide to sata to bypass bug https://marc.info/?t=168129593400001&r=1&w=2

**Review by mario-campos — APPROVED** [review]

File: openbsd-7.3/.pkr.hcl [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+
+  provisioner "file" {
+    source      = "${path.root}/authorized_keys"
+    destination = ".ssh/authorized_keys"
+  }
+
+  # Disable unnecessary services to save CPU cycles.
+  provisioner "shell" {
+    inline = [
+      "rcctl disable check_quotas cron dhcpleased library_aslr ntpd pf pflogd resolvd slaacd smtpd sndiod syslogd",
```
Comment (moritzbuhl) [inline openbsd-7.3/.pkr.hcl:60]: remove dhcpleased, resolvd from here

File: openbsd-7.3/.pkr.hcl [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+    ]
+  }
+
+  provisioner "shell" {
+    inline = ["pkg_add git got cmake meson autoconf-2.71 automake-1.16.5 libtool"]
+  }
+
+  # Run dhclient instead of running dhcpleased; save some CPU cycles.
+  provisioner "shell" {
+    inline = ["echo '!dhclient \\$if' > /etc/hostname.vio0"]
```
Comment (moritzbuhl) [inline openbsd-7.3/.pkr.hcl:70]: remove this

File: openbsd-7.3/.pkr.hcl [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+    inline = ["sed -i 's|/usr/libexec/reorder_kernel &|rdate time.cloudflare.com|' /etc/rc"]
+  }
+
+  # syspatch(8) may sometimes "fail" (with an exit code of 2) if it must update
+  # itself first. No worries -- just gotta run it again.
+  provisioner "shell" {
+    inline           = ["syspatch"]
+    valid_exit_codes = [0, 2]
+  }
+  provisioner "shell" {
```
Comment (moritzbuhl) [inline openbsd-7.3/.pkr.hcl:85]: why is it here twice

File: openbsd-7.3/Vagrantfile [hunk 4]
```diff
@@ -0,0 +1,26 @@
+Vagrant.configure("2") do |config|
+  config.vm.guest = "openbsd"
+
+  config.vm.box = "emulate/openbsd-7.3"
+  config.vm.ignore_box_vagrantfile = true
+  config.vm.box_check_update = false
+  config.vm.box_download_checksum_type = "sha256"
+  config.vm.box_download_checksum = "565c5b324273436403aa72cc55c7461ce513e97c2da45b8da818a1979741eaa0"
```
Comment (moritzbuhl) [inline openbsd-7.3/Vagrantfile:8]: needs a manual build and change

File: openbsd-7.3/Vagrantfile [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+  config.vm.box_url = [
+    "https://app.vagrantup.com/emulate/boxes/openbsd-7.3/versions/0.1.1/providers/virtualbox.box",
```
[the remark this answers is not in the document]
Comment (jimh-gh) [reply openbsd-7.3/Vagrantfile:10]: hash has to be fixed with with mario's builds produces and uploaded to vagrant.

[the remark this answers is not in the document]
Comment (mario-campos) [reply openbsd-7.3/Vagrantfile:10]: Yeah, this URL doesn't exist yet. I'll update them once I've uploaded it.

**Pull request merged**